python/tank/platform/application.py

```diff
@@ -207,6 +209,17 @@ def log_exception(self, msg):
     ##########################################################################################
     # internal API
 
+    def event(self, event):
```

Comment:
this should probably be in the bundle base class so that it works for engines, frameworks and applications.